scripts/badger/bribe_ecosystems.py

```diff
@@ -161,4 +164,11 @@ def bribe_balancer(gauge, mantissa):
             mantissa,  # uint256 amount
         )
 
+    if bribes["bunni"] > 0:
+        prop = web3.solidityKeccak(["address"], [r.bunni.badger_wbtc_bunni_gauge])
```

Comment:
```suggestion
        # TODO: verify this is the right way to generate the prop hash
        prop = web3.solidityKeccak(["address"], [r.bunni.badger_wbtc_bunni_gauge])
```

Reply:
confirmed the prop id is correct against: https://api.hiddenhand.finance/proposal/bunni ✅